bin/handler-twiliosms.rb

```diff
@@ -37,14 +37,13 @@ def handle
     raise 'Please define a valid set of SMS recipients to use this handler' if candidates.nil? || candidates.empty?
 
     recipients = []
-    candidates.each do |mobile, candidate|
-      next unless (candidate['sensu_roles'].include?('all') ||
-          (@event['check']['name'].eql?('keepalive') && candidate['sensu_roles'].include?('keepalive')) ||
-          (@event['check']['subscribers'] &&
-            (candidate['sensu_roles'] & @event['check']['subscribers']).size > 0) || # rubocop:disable Style/ZeroLengthPredicate
-          candidate['sensu_checks'].include?(@event['check']['name'])) &&
-                  (candidate['sensu_level'] >= @event['check']['status'])
-      recipients << mobile
+    candidates.each do |mobile|
+     (@event['check']['name'].eql?('keepalive') && candidate['sensu_roles'].include?('keepalive')) ||
```

Comment:
Seems like we should br removing all this logic, previously all this logic was with an `unless` so we should either keep the existing logic with the conditional or remove it entirely.

At this point as no matter what we do its a breaking change so unless there is something very specific to this handlers filtering that makes sense (as a twillio specific handler) we should rely on native sensu filters being done outside the handler itself. If its accomplished via a filter (as an extension or config) the logic would run within the sensu process and such we can avoid a process fork unless we actually need it.